### ISSUE TITLE
Group Node with my settings

### ISSUE THREAD

**Issue by Aljnk** [comfyanonymous/ComfyUI#3321]
After converting my nodes to "Group Node", it adds them with default settings.

<img width="881" alt="Screenshot_1" src="https://github.com/ltdrdata/ComfyUI-Manager/assets/11256478/246769f4-f6da-4db0-89b0-1acffe8bca6c">

First node - must be. Second - when I add a new one.

**Is it possible to save the settings as well?**

**Comment by Ekundayo39283** [comment]
Yes, it's possible to save the settings of your Group Nodes. After converting your nodes to Group Nodes and adjusting the settings as desired, you can save the changes by either saving the project or exporting the settings separately.

**Comment by Aljnk** [comment]
There are no such options in the right-click menu

<img width="402" alt="Screenshot_1" src="https://github.com/comfyanonymous/ComfyUI/assets/11256478/61546765-dd43-4d9e-9943-a31d9d30da0f">

Save as Component - not save settings - tested.

**Comment by Ekundayo39283** [comment]
Saving settings for Group Nodes might not be directly available through the right-click menu. However, you can still achieve this by using the following workaround:

-  After adjusting the settings of your Group Node(s), take note of all the settings you've modified.
- Go to the main menu or toolbar of ComfyUI and look for an option like "Export Settings" or "Save Settings".
-  Click on this option and follow the prompts to save the settings file to your desired location on your computer.

Alternatively, you can manually apply the same settings to other Group Nodes by adjusting them individually based on the changes you made previously.
